test/wordlist-browser-test.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const assert = require('bsert');
+const wordlistBrowser = require('../lib/hd/wordlist-browser');
+const words = require('../lib/hd/words');
```

Comment:
I think what needs to be actually compared here is `wordlist-browser.js` and `wordlist.js`. They should have identical APIs (a single `get()`) and return the same data.

All the `*-browser.js` files in bcoin are actually not used by bcoin in the nodejs environment. There is a list of `"browser"` aliases in `package.json` that instruct bundler programs like browserify or bpkg to switch out certain files when compiling for use in the browser.

Our test runner `bmocha` does have a headless chrome option (`-H`) to test JS in the browser, first by bundling bcoin using `browserify` which does the alias swapping as it compiles the payload for chrome.

I think if we are going to do more browser tests in bcoin we could organize it differently, lets keep discussing